shared/aux.go

```diff
@@ -352,3 +357,27 @@ func fileExists(files []os.DirEntry, workload string) bool {
 
 	return false
 }
+
+// VerifyFileMatchWithPath verify actual file list matches the expected file list
+func VerifyFileMatchWithPath(actualFileList []string, expectedFileList []string, path string) {
```

Comment:
this func should return err , since you have a conditional 
```
equalLength := Expect(len(actualFileList)).To(Equal(len(expectedFileList)))
	if !equalLength {
		LogLevel("error", fmt.Sprintf("actual file and expected file lists size are NOT equal: "+
			"%d vs %d", len(actualFileList), len(expectedFileList)))
	}
```

and the caller deals with it how they want

Reply:
fixed